sw/device/tests/BUILD

```diff
@@ -516,6 +519,7 @@ opentitan_functest(
     srcs = ["otbn_ecdsa_op_irq_test.c"],
     verilator = verilator_params(
         timeout = "long",
+        tags = ["slow_in_verilator"],
```

Comment:
There's already a notion of expected test time. Can we use bazel's test timeouts or sizes?